sns_ik_lib/include/sns_ik/sns_velocity_ik.hpp

```diff
@@ -39,15 +39,7 @@ namespace sns_ik {
 
 struct Task {
     MatrixD jacobian;  //!< the task Jacobian
-    MatrixD jacobian_derivate;  //!< time derivartive of the Jacobian, needed only at acceleration control level
-    VectorD desired;  //!< desired velocity/acceleration
-    Scalar h;
-    VectorD q0;
-    Scalar h0;
-    Scalar min;  //!< min for the constraints
-    Scalar max;  //!< max for the constraints
-    Scalar value;  //!< value for the constraints
-    int is_constraints;
+    VectorD desired;   //!< desired velocity in task space
```

Comment:
StackOfTasks is used all over the code. Maybe we should open a separate ticket to change it.